package.json

```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0-beta.1 -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@11.0.0-beta.7 -W  && git checkout -- package.json",
```

Comment:
We stopped shipping `cdkv2` tag (which `^` would resolve...) therefore pinning version here.